product/views.py

```diff
@@ -1,3 +1,250 @@
-from django.shortcuts import render
+from datetime      import datetime
+from mypage.models import Review, ReviewImage
 
-# Create your views here.
+from django.http                  import JsonResponse, request
+from django.views                 import View
+from django.db.models.query_utils import Q
+from django.db.models             import Count, Avg
+
+from product.models import (
+    Category, Menu, Product, ProductImage)
+from mypage.models  import Favorite 
+
+class MainCategoryView(View):
+    def get(self, request):
+        
+        menus    = Menu.objects.all()
+        menuList = []
```

Comment:
자바스크립트 아니고 파이썬 컨벤션에 맞게 카멜케이스 쓰지 마세요.
```suggestion
        menu_list = []
```

Reply:
넵 리뷰 감사드립니다. 바로 수정하겠습니다!